src/main/java/it/pagopa/transactions/controllers/v1/TransactionsController.java

```diff
@@ -180,7 +185,21 @@ public Mono<ResponseEntity<TransactionInfoDto>> updateTransactionAuthorization(
                                         transactionId,
                                         transactionIdDecoded
                                 )
-                        )
+                        ).map(updateAuthorizationRequest -> {
+                            TransactionId domainTransactionId = new TransactionId(transactionIdDecoded);
+                            ExclusiveLockDocument lockDocument = new ExclusiveLockDocument(
+                                    "PATCH-auth-request-%s".formatted(domainTransactionId.value()),
+                                    "transactions-service"
+                            );
+                            boolean locked = exclusiveLockDocumentWrapper.saveIfAbsent(
+                                    lockDocument
+                            );
+                            log.info("UpdateTransactionAuthorization lock acquired for transactionId: [{}] with key: [{}]: [{}]", domainTransactionId.value(), lockDocument.id(), locked);
+                            if (!locked) {
+                                throw new LockNotAcquiredException(domainTransactionId, lockDocument);
+                            }
```

Comment:
```suggestion
                            boolean lockAcquired = exclusiveLockDocumentWrapper.saveIfAbsent(
                                    lockDocument
                            );
                            log.info("UpdateTransactionAuthorization lock acquired for transactionId: [{}] with key: [{}]: [{}]", domainTransactionId.value(), lockDocument.id(), lockAcquired);
                            if (!lockAcquired) {
                                throw new LockNotAcquiredException(domainTransactionId, lockDocument);
                            }
```

Maybe a bit clearer, rn `locked` is ambiguous if it means "already locked before" or "lock acquired here"